<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhaseWPHabituation</class>
 <widget class="QWizardPage" name="PhaseWPHabituation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Habituation phase type</string>
  </property>
  <property name="subTitle">
   <string>Enter details for a habituation phase</string>
  </property>
  <attribute name="pageId">
   <string notr="true">3</string>
  </attribute>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Maximum number of trials:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinboxHabNTrials"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Habituation type:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbHabReduction">
        <property name="toolTip">
         <string>Looking time decreases to a specified level</string>
        </property>
        <property name="text">
         <string>Looking time reduction criteria</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbHabAccum">
        <property name="toolTip">
         <string>Subject must accumulate a specified amount of looking time over the entire phase</string>
        </property>
        <property name="text">
         <string>Accumulated looking time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
